shaders/Skinning.vert

```diff
@@ -35,9 +35,9 @@ void main () {
     normal += (bones[vertexBoneIDs[2]] * vec4(vertexNormal, 0.0)) * vertexWeights[2];
     normal += (bones[vertexBoneIDs[3]] * vec4(vertexNormal, 0.0)) * vertexWeights[3];
     
-    gl_Position = viewProjection * (model * position);
-    vertexOut.pos = (model * position).xyz;
-    vertexOut.normal = normalize(normalMatrix * normal.xyz);
```

Comment:
The `normalMatrix` uniform is no longer used, so the variable could be removed.

Reply:
@Ittis  is looking into this.